package.json

```diff
@@ -59,7 +59,7 @@
     "redis": "^2.8.0",
     "redis-clustr": "^1.6.0",
     "request-ip": "^2.1.1",
-    "spid-passport": "git://github.com/gunzip/spid-passport.git#681554025efd2",
+    "spid-passport": "git://github.com/teamdigitale/spid-passport.git",
```

Comment:
I've created the `v1.1.0` tag https://github.com/teamdigitale/spid-passport/tree/v1.1.0